corehq/motech/repeaters/management/commands/list_repeater_domain_admins.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import logging
+from argparse import FileType
+
+from django.core.management.base import BaseCommand
+from corehq.apps.users.models import WebUser
+from corehq.motech.repeaters.models import Repeater
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """Shows names and admin emails of Domains with Repeaters."""
+
+    def add_arguments(self, parser):
+        parser.add_argument("--csv", action="store_true", default=False,
+            help="Write output as CSV data instead of padded table.")
+        parser.add_argument("-o", "--output", metavar="FILE", type=FileType("w"),
+            default=self.stdout, help="Write output to %(metavar)s (default=STDOUT).")
+
+    def handle(self, *args, **options):
+        log.debug("fetching repeaters")
```

Comment:
for management commands I feel it's best to use `self.stdout.write`. Are you purposely wanting these outputs to go to logs/athena?